### ISSUE TITLE
utils.music.get_time_maps_from_alignment removed from API without updating partitura.readthedocs.io

### ISSUE THREAD

**Issue by eoinroe** [CPJKU/partitura#348]
The example code from [here](https://partitura.readthedocs.io/en/latest/Tutorial/notebook.html#5.-Handling-Alignment-Information-(Match-files)) has not been updated so when running it with the latest version of Partitura it produces the following error: 

`AttributeError: module 'partitura.utils.music' has no attribute 'get_time_maps_from_alignment'`

It appears that `get_time_maps_from_alginment` has been completely removed from the API instead of being deprecated so it is unclear which method to use in its place.

**Comment by neosatrapahereje** [comment]
Good catch! The method was moved to the `performance_codec` module (in  `musicanalysis`) and can be accessed as follows:


```python
from partitura.musicanalysis.performance_codec import get_time_maps_from_alignment
```

We'll update the tutorial and have a deprecation notification in `utils.music` to fix this issue.

**Comment by eoinroe** [comment]
Thanks for your swift reply! I am now able to run the code from the tutorial and recreate the plot using match files from the Vienna 4x22 Piano Corpus.

Cheers!
